draft-pauly-dprive-oblivious-doh.md

```diff
@@ -164,20 +170,28 @@ Clients MUST set the HTTP Content-Type header to "application/oblivious-dns-mess
 to indicate that this request is an Oblivious DoH query intended for proxying. Clients
 also SHOULD set this same value for the HTTP Accept header.
 
-Proxies MUST check that Client requests are correctly encoded, and MUST return a
-4xx (Client Error) if the check fails, along with the Proxy-Status response header
-with an "error" parameter of type "http_request_error" {{!I-D.ietf-httpbis-proxy-status}}.
 A correctly encoded request has the HTTP Content-Type header "application/oblivious-dns-message",
 uses the HTTP POST method, and contains "targethost" and "targetpath" variables.
+The Proxy MUST validate these parameters and construct a URI Template as follows:
```

Comment:
Proxies don't construct a URI template, they just construct an HTTP request

Reply:
They use the URI Template to generate the request. I tried to clarify.